profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/tmgmt_dgt_connector.inc

```diff
@@ -244,3 +244,24 @@ function _tmgmt_dgt_connector_translator_access($op, array $translator, array $a
 
   return tmgmt_translator_access($op, $translator, $account);
 }
+
+/**
+ * Check if 'Send to cart' is possible.
+ *
+ * @param object $entity
+ *   The entity to check.
+ * @param array $cart
+ *   The cart to be generated.
+ *
+ * @return bool
+ *   Returns true if 'Send to cart' is possible, false otherwise.
+ */
+function _tmgmt_dgt_connector_translation_possible($entity, array $cart) {
```

Comment:
The name of the function is a bit confusing it doesn't check if translation is possible, but rather if sending to cart is possible.